plugins/evernote.koplugin/main.lua

```diff
@@ -1,22 +1,23 @@
 local BD = require("ui/bidi")
-local InputContainer = require("ui/widget/container/inputcontainer")
-local LoginDialog = require("ui/widget/logindialog")
-local InfoMessage = require("ui/widget/infomessage")
-local NetworkMgr = require("ui/network/manager")
+local DEBUG = require("dbg")
 local DataStorage = require("datastorage")
-local DocSettings = require("docsettings")
-local UIManager = require("ui/uimanager")
-local Screen = require("device").screen
-local util = require("ffi/util")
 local Device = require("device")
-local DEBUG = require("dbg")
+local DocSettings = require("docsettings")
+local InfoMessage = require("ui/widget/infomessage")
+local InputContainer = require("ui/widget/container/inputcontainer")
 local JoplinClient = require("JoplinClient")
-local T = require("ffi/util").template
+local LoginDialog = require("ui/widget/logindialog")
+local MyClipping = require("clip")
 local _ = require("gettext")
 local N_ = _.ngettext
-local slt2 = require('slt2')
-local MyClipping = require("clip")
+local NetworkMgr = require("ui/network/manager")
+local Screen = require("device").screen
+local T = require("ffi/util").template
+local UIManager = require("ui/uimanager")
```

Comment:
This one a little higher. ;-)

Reply:
Are there any issues blocking this PR?

Reply:
Well yes, the requires. Like I wrote above:

> Your previous PR was perfectly fine except for the minor whitespace issue with using two spaces instead of four. The require thing was just a parenthetical remark. Also because touching those lines will necessitate me to do a lot of nitpicking (like some of the naming schemes, the one require randomly using single quotes, etc.)

Reply:
Let me know what you think now. I have removed all changes in requires apart from adding json library